src/platforms/apple/common/enriching-events/user-feedback.mdx

```diff
@@ -13,7 +13,7 @@ import Sentry
 
 let eventId = SentrySDK.capture(message: "My message.")
 
-let userFeedback = UserFeedack(eventId: eventId)
+let userFeedback = UserFeedback(eventId: eventId)
```

Comment:
The name is actually wrong on the `SWIFT_NAME`: https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/Public/SentryUserFeedback.h#L11

Reply:
Code needs fixing first.

```suggestion
let userFeedback = UserFeedack(eventId: eventId)
```

Reply:
https://github.com/getsentry/sentry-cocoa/pull/829/